tracim/development.ini.base

```diff
@@ -35,6 +35,7 @@ full_stack = true
 # You can set french as default language by uncommenting next line
 # lang = fr
 cache_dir = %(here)s/data
+preview_cache_dir = /tmp/tracim/cache/previews/
```

Comment:
ok for the preview cache.
Now : where are stored file depot contents ? Is it parameterized ?